logging failures

FAILED test_app/tests/authentication/authenticator_plugins/test_radius.py::test_authenticator_plugin

```
E           AssertionError: expected call not found.
E           Expected: authenticate(username='user-73aa1c', password='radius_password-4aa69d')
E           Actual: authenticate(None, 'user-73aa1c', 'radius_password-4aa69d')
```


FAILED test_app/tests/authentication/authenticator_plugins/test_oidc.py::test_user_data

```
E           AssertionError: expected call not found.
E           Expected: decode('token', 'VALUE', 'VALUE', 'VALUE')
E           Actual: decode('token', key='-----BEGIN PUBLIC KEY-----\nVALUE\n-----END PUBLIC KEY-----', algorithms='VALUE', audience='VALUE')
```

--- a/test_app/tests/authentication/authenticator_plugins/test_oidc.py
+++ b/test_app/tests/authentication/authenticator_plugins/test_oidc.py
@@ -146,7 +146,7 @@ def test_user_data(mockedrequest, mockeddecode, mocksetting):
     mockeddecode.return_value = mr.json()
     data = ap.user_data("token")
 
-    assert mockeddecode.called_with("token", "VALUE", "VALUE", "VALUE")
+    mockeddecode.assert_called_once_with("token", "VALUE", "VALUE", "VALUE")
     assert "key" in data
 
     # Decode failure

--- a/test_app/tests/authentication/authenticator_plugins/test_oidc.py
+++ b/test_app/tests/authentication/authenticator_plugins/test_oidc.py
@@ -146,7 +146,7 @@ def test_user_data(mockedrequest, mockeddecode, mocksetting):
     mockeddecode.return_value = mr.json()
     data = ap.user_data("token")
 
-    mockeddecode.assert_called_once_with("token", "VALUE", "VALUE", "VALUE")
+    mockeddecode.assert_called_once_with('token', key='-----BEGIN PUBLIC KEY-----\nVALUE\n-----END PUBLIC KEY-----', algorithms='VALUE', audience='VALUE')
     assert "key" in data
 
     # Decode failure

--- a/test_app/tests/authentication/authenticator_plugins/test_radius.py
+++ b/test_app/tests/authentication/authenticator_plugins/test_radius.py
@@ -85,7 +85,7 @@ def test_authenticator_plugin(backend_cls, unauthenticated_api_client, radius_co
         "RADIUS_PORT": radius_configuration["PORT"],
         "RADIUS_SECRET": radius_configuration["SECRET"],
     }
-    backend.authenticate.assert_called_once_with(username=random_username, password=random_password)
+    backend.authenticate.assert_called_once_with(None, random_username, random_password)
     assert AuthenticatorUser.objects.filter(uid=random_username, provider=radius_authenticator).exists()
     assert User.objects.filter(username=random_username).exists()
 

--- a/test_app/tests/authentication/authenticator_plugins/test_radius.py
+++ b/test_app/tests/authentication/authenticator_plugins/test_radius.py
@@ -85,7 +85,7 @@ def test_authenticator_plugin(backend_cls, unauthenticated_api_client, radius_co
         "RADIUS_PORT": radius_configuration["PORT"],
         "RADIUS_SECRET": radius_configuration["SECRET"],
     }
-    backend.authenticate.called_once_with(username=random_username, password=random_password)
+    backend.authenticate.assert_called_once_with(username=random_username, password=random_password)
     assert AuthenticatorUser.objects.filter(uid=random_username, provider=radius_authenticator).exists()
     assert User.objects.filter(username=random_username).exists()
 

